stable/prometheus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 1.3.1
+version: 1.4.0
```

Comment:
My intent is to have the the chart track the Prometheus version, so I think a semver (`+r1`) build tag will be sufficient. I have a PR pending for v1.4.0.